lib/cucumber/runtime.rb

```diff
@@ -30,8 +30,12 @@ def initialize(original_exception, path)
   class FileNotFoundException < FileException
   end
 
-  class FeatureFolderNotFoundException < FileException
-    include FixRuby21Bug9285 if Cucumber::RUBY_2_1 || Cucumber::RUBY_2_2 || Cucumber::RUBY_2_3
+  class FeatureFolderNotFoundException < Exception
+    attr :path
+    def initialize(path)
```

Comment:
@nodo I thought previously FeatureFolderNotFoundException had the getter for `path` variable, because it had extended `FileException`. But as we do not use the getter right now, removing it might be better. I am going to remove it.